lib/console/alerts/alert_events.ex

```diff
@@ -78,16 +81,17 @@ defmodule Console.AlertEvents do
           if not webhook_restrict_to_prevent_flapping do
             webhook_attrs = Map.put(attrs, :type, "webhook")
             if Map.has_key?(alert_event_webhook_config, "value") do
+              # since email and webhook can have diff value, must check against buffer value checked
               if alert_event_webhook_config["value"] == details.buffer do
                 create_alert_event(webhook_attrs)
+                Alerts.update_alert_last_triggered_at(alert)
               end
             else
               create_alert_event(webhook_attrs)
+              Alerts.update_alert_last_triggered_at(alert)
```

Comment:
this technically could update twice (once for email and once for webhook) but better than to update when it has not created a new alert event bc of flapping prevention